src/engine/bank03.asm

```diff
@@ -3283,279 +3272,158 @@ Script_DrMason: ; d727 (3:5727)
+	do_frames 40
+	print_text_string Text05e6
+	close_text_box
+	move_player WEST, 1
+	move_player WEST, 1
+	set_player_direction SOUTH
+	move_player SOUTH, 1
+	move_player SOUTH, 1
+	move_player SOUTH, 1
+	set_player_direction WEST
+	move_active_npc NPCMovement_d894
+	print_text_string Text05e7
+	set_dialog_npc $07
+	print_text_string Text05e8
 
 .ows_d7bc
-	run_command ScriptCommand_CloseTextBox
-	run_command ScriptCommand_ShowSamTutorialMultichoice
-	run_command ScriptCommand_CloseTextBox
-	run_command ScriptCommand_JumpIfFlagEqual
-	db EVENT_FLAG_75
-	db $07
-	dw .ows_d80c
-	run_command ScriptCommand_JumpIfFlagEqual
-	db EVENT_FLAG_75
-	db $01
-	dw .ows_d7e8
-	run_command ScriptCommand_JumpIfFlagEqual
-	db EVENT_FLAG_75
-	db $02
-	dw .ows_d7ee
-	run_command ScriptCommand_JumpIfFlagEqual
-	db EVENT_FLAG_75
-	db $03
-	dw .ows_d7f4
-	run_command ScriptCommand_JumpIfFlagEqual
-	db EVENT_FLAG_75
-	db $04
-	dw .ows_d7fa
-	run_command ScriptCommand_JumpIfFlagEqual
-	db EVENT_FLAG_75
-	db $05
-	dw .ows_d800
-	run_command ScriptCommand_JumpIfFlagEqual
-	db EVENT_FLAG_75
-	db $06
-	dw .ows_d806
-	run_command ScriptCommand_PrintTextString
-	tx Text05d6
-	run_command ScriptCommand_Jump
-	dw .ows_d7bc
+	close_text_box
+	show_sam_tutorial_multichoice
+	close_text_box
+	jump_if_flag_equal EVENT_FLAG_75, $07, .ows_d80c
+	jump_if_flag_equal EVENT_FLAG_75, $01, .ows_d7e8
+	jump_if_flag_equal EVENT_FLAG_75, $02, .ows_d7ee
+	jump_if_flag_equal EVENT_FLAG_75, $03, .ows_d7f4
+	jump_if_flag_equal EVENT_FLAG_75, $04, .ows_d7fa
+	jump_if_flag_equal EVENT_FLAG_75, $05, .ows_d800
+	jump_if_flag_equal EVENT_FLAG_75, $06, .ows_d806
+	print_text_string Text05d6
+	script_jump .ows_d7bc
 
 .ows_d7e8
-	run_command ScriptCommand_PrintTextString
-	tx Text05d7
-	run_command ScriptCommand_Jump
-	dw .ows_d7bc
+	print_text_string Text05d7
+	script_jump .ows_d7bc
 
 .ows_d7ee
-	run_command ScriptCommand_PrintTextString
-	tx Text05d8
-	run_command ScriptCommand_Jump
-	dw .ows_d7bc
+	print_text_string Text05d8
+	script_jump .ows_d7bc
 
 .ows_d7f4
-	run_command ScriptCommand_PrintTextString
-	tx Text05d9
-	run_command ScriptCommand_Jump
-	dw .ows_d7bc
+	print_text_string Text05d9
+	script_jump .ows_d7bc
 
 .ows_d7fa
-	run_command ScriptCommand_PrintTextString
-	tx Text05da
-	run_command ScriptCommand_Jump
-	dw .ows_d7bc
+	print_text_string Text05da
+	script_jump .ows_d7bc
 
 .ows_d800
-	run_command ScriptCommand_PrintTextString
-	tx Text05db
-	run_command ScriptCommand_Jump
-	dw .ows_d7bc
+	print_text_string Text05db
+	script_jump .ows_d7bc
 
 .ows_d806
-	run_command ScriptCommand_PrintTextString
-	tx Text05dc
-	run_command ScriptCommand_Jump
-	dw .ows_d7bc
+	print_text_string Text05dc
+	script_jump .ows_d7bc
 
 .ows_d80c
-	run_command ScriptCommand_PrintTextString
-	tx Text05e9
-	run_command ScriptCommand_AskQuestionJumpDefaultYes
-	dw 0000
-	dw .ows_d817
-	run_command ScriptCommand_Jump
-	dw .ows_d7bc
+	print_text_string Text05e9
+	ask_question_jump_default_yes 0000, .ows_d817
```

Comment:
We could introduce a `NULL` constant to use in place of this `0000`. (Also in that case, probably just get rid of `NO_JUMP` in favor of `NULL`.)